game/src/edit/traffic_signals/edits.rs

```diff
@@ -46,7 +52,7 @@ impl ChangeDuration {
                         "fixed",
                         "adaptive",
                         None,
-                        match current {
+                        match signal.stages[idx].phase_type {
                             PhaseType::Fixed(_) => true,
                             PhaseType::Adaptive(_) => false,
                         },
```

Comment:
How about we add `Line("Minimum time required for crosswalk").secondary().draw(ctx)` right above the "Apply" button, to explain to the player why different signals have different minimums in the spinner?

Reply:
That's a good idea!
I did consider doing something like this, but I was unsure of the best way to achieve it, so thanks!